seed/lib/mappings/data/pm-mapping.json

```diff
@@ -152,6 +206,60 @@
     "type": "float",
     "schema": ""
   },
+  {
+    "display_name": "Home Energy Score ID",
+    "to_field": "home_energy_score_id",
+    "to_table_name": "PropertyState",
+    "from_field": "Home Energy Score ID",
+    "units": "",
+    "type": "string",
+    "schema": ""
+  },
+  {
+    "display_name": "Jurisdiction Property Lot ID",
+    "to_field": "jurisdiction_property_lot_id",
+    "to_table_name": "PropertyState",
+    "from_field": "Jurisdiction Property Lot ID",
+    "units": "",
+    "type": "string",
+    "schema": ""
+  },
+  {
+    "display_name": "Latitude",
```

Comment:
I just checked and espm doesn't repot lat/lng. I wonder if we should still keep this though. @nllong thoughts?